app/javascript/pages/browse/MapBrowser.vue

```diff
@@ -17,119 +17,169 @@
       @map-click:points="clicked"
       @geolocate-error="geolocateError"
       @geolocate-geolocate="geolocate"
+      @map-init="initialized"
+      v-if="contributions.length > 0"
     />
+
+    <div v-if="contributions.length === 0">
+      <p style="text-align: center;">No match found.</p>
+    </div>
   </section>
 </template>
 
 <script>
-  import Mapbox from 'mapbox-gl-vue'
-  const mbxClient = require('@mapbox/mapbox-sdk');
-  const geoCoding = require('@mapbox/mapbox-sdk/services/geocoding');
-  //  FIXME Attempt to pass accessToken in script tag as seen in line 6
-  const baseClient = mbxClient({ accessToken: "pk.eyJ1IjoibXV0dWFsLWFpZC1hcHAiLCJhIjoiY2tmZTBvd3UwMDBhbTJ4cDlic2JmMWZoaiJ9.rWscBjdl1SMT5N0yekIJYg"});
-  const geoCodingService = geoCoding(baseClient)
+import Mapbox from 'mapbox-gl-vue'
+const mbxClient = require('@mapbox/mapbox-sdk')
```

Comment:
I assume you've got an auto-formatter or something like that. Because it unindented all the code in this `script` section, we're having trouble seeing what did and did not change. Can you revert the whitespace changes so it's easier for us to review this PR?

Reply:
Done :+1: